v_4_0_0/master_template.go

```diff
@@ -198,7 +198,7 @@ systemd:
       Wants=k8s-setup-network-env.service k8s-setup-kubelet-config.service
       After=k8s-setup-network-env.service k8s-setup-kubelet-config.service
       Description=k8s-kubelet
-      StartLimitIntervalSec=0
+      StartLimitIntervalSec=30
```

Comment:
This is here to overcome issue https://github.com/kubernetes/kubernetes/issues/71078
I hope on the time of release it will be solved or we can dig more to find out the reason for such behaviour